### ISSUE TITLE
Issue with k8s.io/docs/tutorials/hello-minikube/

### ISSUE THREAD

**Issue by BorntraegerMarc** [kubernetes/website#18658]
**This is a Bug Report**

**Problem:**
The docs want me to enable the add-on heapster. But this add-on is not maintained anymore and when I was running the commands I got the error code `enable failed: property name "heapster" not found`

**Proposed Solution:**

**Page to Update:**
https://kubernetes.io/docs/tutorials/hello-minikube/

Latest minikube

**Comment by Kartik494** [comment]
@BorntraegerMarc yes minikube add on heapster is deprecated now ,so how would you like to update doc!
https://github.com/kubernetes-retired/heapster

Either we can update it with another example like minikube addons enable "add on name".
Or we can remove heapster from add on and add new list of add ons.

**Comment by gavinfish** [comment]
Execute `minikube addons list` on the latest minikube version 1.6.2 and I get this:

```
- addon-manager: enabled
- dashboard: disabled
- default-storageclass: enabled
- efk: disabled
- freshpod: disabled
- gvisor: disabled
- helm-tiller: disabled
- ingress: disabled
- ingress-dns: disabled
- logviewer: disabled
- metrics-server: disabled
- nvidia-driver-installer: disabled
- nvidia-gpu-device-plugin: disabled
- registry: disabled
- registry-creds: disabled
- storage-provisioner: enabled
- storage-provisioner-gluster: disabled
```

**Comment by BorntraegerMarc** [comment]
Don't know how it should be updated. I'm a total newbie :)

I guess if you want to portrait the add ons in this tutorial then any working add on will do the job.

**Comment by php-coder** [comment]
It seems like the same as #18420

**Comment by Kartik494** [comment]
@BorntraegerMarc i have create the PR for the same and it's get merged.
Thanks

**Comment by php-coder** [comment]
The issue was fixed by #18673
/close
